common/lanelet2_extension/include/lanelet2_extension/regulatory_elements/BusStopRule.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+/*
+ * Copyright (C) 2023 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+#include <lanelet2_core/primitives/RegulatoryElement.h>
+#include <boost/algorithm/string.hpp>
+#include <unordered_set>
+
+namespace lanelet
+{
+/**
+ * @brief Represents a virtual bus stop and wait line horizontally laying on the roadway. It indicates whether a given participant bus 
+ * should stop and wait momentarily before passing the line. General usage is as a  bus stop line that is not represented by an actual
+ * physical roadway object. By default, it only apply to bus in the participant list.
+ *
+ * A BusStopRule is created from a list of contiguous LineString3d and participant bus which should stop and wait at bus stop before crossing.
+ * The object is agnostic to the line's invertedness.
+ *
+ * @ingroup RegulatoryElementPrimitives
+ * @ingroup Primitives
+ */
+class BusStopRule : public StopRule
+{
+public:
+  static constexpr char RuleName[] = "stop_rule";
+  static constexpr char Participants[] = "participants";
+  std::unordered_set<std::string> participants_ = ["bus"];
```

Comment:
Please make sure this compiles. I think correct initialization syntax is different.
https://www.techiedelight.com/initialize-set-unordered_set-cpp/